app/assets/images/payment_icons/sofort.svg

```diff
@@ -1 +1 @@
\ No newline at end of file
+<svg width="38" height="24" viewBox="0 0 38 24" fill="none" xmlns="http://www.w3.org/2000/svg" role="img" aria-labelledby="pi-sofort"><title id="pi-sofort">SOFORT</title><path fill="#F5F5F5" d="M0 0h38v24H0z"/><path d="M35.185 20.8H0L2.674 3.2H38l-2.815 17.6z" fill="#393A41"/><path d="M11.671 9.28c-1.58 0-2.78 1.353-2.78 2.965 0 1.458.984 2.475 2.325 2.475 1.58 0 2.772-1.391 2.772-3.003.009-1.468-.976-2.437-2.317-2.437zm-.355 4.068c-.605 0-1.027-.461-1.027-1.171 0-.787.522-1.534 1.275-1.534.604 0 1.026.48 1.026 1.19.008.796-.521 1.515-1.275 1.515zM21.18 9.28c-1.58 0-2.772 1.353-2.772 2.965 0 1.458.985 2.475 2.326 2.475 1.58 0 2.773-1.391 2.773-3.003.008-1.468-.977-2.437-2.327-2.437zm-.347 4.068c-.604 0-1.026-.461-1.026-1.171 0-.787.521-1.534 1.275-1.534.603 0 1.026.48 1.026 1.19 0 .796-.521 1.515-1.276 1.515h.001zm7.325-2.35c0-.98-.654-1.603-1.846-1.603H24.64l-.736 5.21h1.407l.232-1.641h.066l.744 1.641h1.614l-1.034-1.824c.77-.307 1.224-.968 1.224-1.783h.001zm-2.251.864H25.7l.157-1.095h.19c.439 0 .67.154.67.49 0 .394-.33.605-.81.605zm-18.63-.385c-.538-.317-.655-.394-.655-.567 0-.21.215-.316.505-.316.34 0 .853.048 1.358.604A3.63 3.63 0 019.08 9.99c-.71-.47-1.374-.71-1.977-.71-1.2 0-1.895.748-1.895 1.65 0 .778.504 1.18 1.1 1.536.538.316.67.422.67.614 0 .211-.223.326-.521.326-.513 0-1.11-.374-1.465-.758l-.207 1.449a2.52 2.52 0 001.697.624c1.241 0 1.912-.73 1.912-1.661 0-.768-.43-1.18-1.117-1.583zm10.428.201H16.05l.132-.912h1.77a3.538 3.538 0 011.027-1.37h-2.897c-.645 0-1.191.508-1.283 1.199l-.562 4.02h1.407l.232-1.642h1.672l.124-.891c.008-.144.017-.27.034-.404h-.001zM28.19 9.396c.38.336.612.806.662 1.371h1.175l-.537 3.838h1.406l.538-3.838h1.308l.19-1.372h-4.742z" fill="#EDEDED"/></svg>
```

Comment:
- The icon missing the border. [video demonstration](https://screenshot.click/07-59-m2etv-dlw1l.mp4)
- Please remove the grey background `<path fill="#F5F5F5" d="M0 0h38v24H0z"/>`

The [guidelines](https://github.com/activemerchant/payment_icons/blob/master/CONTRIBUTING.md#template) provided an example that you can take from.

If I try to add it myself, the icon's branding background color goes through the border. You might want to resize down the icon further.

Reply:
Updated.